regolith/schemas.py

```diff
@@ -10,17 +11,20 @@
 
 ACTIVITIES_TYPES = ["teaching", "research"]
 AGENCIES = ["nsf", "doe", "other"]
+ALLOWED_STATI = ["proposed", "started", "finished", "back_burner", "paused", "cancelled"]
```

Comment:
please remove.   These comments may have gotten lost on another PR perhaps, but this is not the right pattern.  You should be importing the `PROJECTA` stati into the progress report.  Don't duplicate them with new non-specific names.